webapp/components/integrations/components/installed_outgoing_webhooks.jsx

```diff
@@ -115,6 +115,7 @@ export default class InstalledOutgoingWebhooks extends React.Component {
                     creator={this.state.users[outgoingWebhook.creator_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
@jwilander was mentioning in the second comment about keeping` installed_incoming_webhook` the way as it exists in PR. Change `installed_incoming_webhooks` to be a connected component with the channel as a selector and pass it as props to `installed_incoming_webhook`.